lib/gui/app/modules/image-writer.js

```diff
@@ -275,21 +311,7 @@ exports.flash = (image, drives) => {
       errorCode: error.code
     })
 
-    if (error.code === 'EVALIDATION') {
-      analytics.logEvent('Validation error', analyticsData)
-    } else if (error.code === 'EUNPLUGGED') {
-      analytics.logEvent('Drive unplugged', analyticsData)
-    } else if (error.code === 'EIO') {
-      analytics.logEvent('Input/output error', analyticsData)
-    } else if (error.code === 'ENOSPC') {
-      analytics.logEvent('Out of space', analyticsData)
-    } else if (error.code === 'ECHILDDIED') {
-      analytics.logEvent('Child died unexpectedly', analyticsData)
-    } else {
-      analytics.logEvent('Flash error', _.merge({
-        error: errors.toJSON(error)
-      }, analyticsData))
-    }
+    handleErrorLogging(error, analyticsData)
```

Comment:
We might get some double-reporting here, if all devices fail over the course of the flash, I think (?)